index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mo&lsquo;athal&lsquo;s Portfolio</title>
+</head>
+
+<body>
+    <div class="container flex">
+        <header class="toolbar">
+            <span class="theme logo"> Mo&lsquo;athal S. Kachi</span>
+            <button></button>
+        </header>
+        <section class="headline flex">
+            <h1 class="mainTxt">Mo&lsquo;athal</h1>
+            <h3 class="spprtTxt">Software Developer</h3>
+        </section>
+    </div>
+</body>
+
+</html>
```

Comment:
- [ ] You are missing a great deal of content. 
### Currently, your page looks like this
![image](https://user-images.githubusercontent.com/45460192/176540151-878bc781-9e3e-4b69-be87-4644ccd5495d.png)
### It should look like this
![image](https://user-images.githubusercontent.com/45460192/176540399-1e04fc0e-c252-4da7-8a07-202a06f361b3.png)